pyproject.toml

```diff
@@ -136,6 +136,12 @@ src_paths = ["src", "examples", "tests"]
 asyncio_mode = "auto"
 required_plugins = [ "pytest-asyncio", "pytest-mock" ]
 
+[tool.mypy]
+# This is a temporary hack, for details see:
+# * https://github.com/frequenz-floss/frequenz-sdk-python/pull/219
+# * https://github.com/frequenz-floss/frequenz-sdk-python/issues/226
+exclude = ["src/frequenz/sdk/timeseries/_formula_engine/"]
```

Comment:
I don't get this. This is excluding the `_formula_engine` subdirectory because it has a lot of errors that need to be fixed in a separate PR. This is not related to tests.

Reply:
Ok, then it is fine.
I thought you wanted to quiet the tests because they are failing randomly.